FILE packages/desktop/electron-builder.yml
RANGE {"path":"packages/desktop/electron-builder.yml","old_lines":[11,15],"new_lines":[11,15],"diff_block_lines":[5,11],"hunk_header":"@@ -11,5 +11,5 @@ extraResources:","substantive_changed_lines":2}
@@ -11,5 +11,5 @@ extraResources:
 publish:
   provider: github
-  owner: anthropics
+  owner: getpaseo
   repo: paseo
 mac:
